drafts/use-cases/1.entry-point.md

```diff
@@ -28,18 +28,27 @@ HTTP 200 OK
 ```
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@id": "/api",
     "@type": "hydra:EntryPoint",
-    "events": {
-        "@id": "/api/events", 
+    "#events": {
+        "@id": "/api/events",
         "title": "List of events",
         "@type": "hydra:Collection",
-        "operation": {
-            "@type": "hydra:Operation",
-            "title": "Create new event",
-            "method": "POST",
-            "expects": "schema:Event"
-        }
+        "operation": [
+            {
+               "@type": ["hydra:Operation", "schema:ViewAction"],
+                "title": "List events",
+                "method": "GET",
```

Comment:
Yes but everything else in Hydra is **explicit, or try at your own peril**. Ie. if `GET` operation is absent, one could assume that `GET` is not supported, right?

I like this way of everything being explicit because you may have resources which are write-only. How would you model that?